pages/01.HDM Documentation/06.vcd/80.Troubleshooting Guide/default.md

```diff
@@ -162,10 +162,11 @@ Following are known limitations with virtual machine disk controller configurati
 
 
 
-1. Virtual machines having SATA and IDE based virtual disks
-2. Virtual machines having more than 3 controllers if the first controller is not paravirtual.
-3. Virtual machines with more than 15 virtual disks for the first paravirtual controller.
-4. Virtual machines with LSI controller as boot disk and paravirtual controller as data disks. 
+1. Virtual machines having IDE and NVMe based virtual disks.
+2. Virtual machines having more than 3 controllers if the first controller is not paravirtual
```

Comment:
Point 2, 3 and 4 limitations have been resolved

Reply:
removed 2,3 &4